packages/blockchain-wallet-v4-frontend/src/assets/locales/de.json

```diff
@@ -1099,7 +1099,7 @@
   "scenes.reset2fa.secondstep.back": "Zurückgehen",
   "scenes.reset2fa.secondstep.continue": "Weiter",
   "scenes.reset2fa.thirdstep.reset": "Zurücksetzen",
-  "scenes.reset2fa.thirdstep.step3": "Schritt 3 von 3",
+  "scenes.reset2fa.thirdstep.step3": "Schritt 2 von 2",
```

Comment:
Your should replace all occurences of scenes.reset2fa.thirdstep.step3 by scenes.reset2fa.thirdstep.step2 to avoid confusion

Reply:
good call